admin-app/src/pages/AdminManagingScreen/AdminManagingScreen.jsx

```diff
@@ -118,7 +119,7 @@ const AdminManageScreen = () => {
         try {
             const updatedAdmin = {
                 email: adminEmail,
-                password: adminPassword // Dodali smo polje za lozinku
+                phoneNumber: null
```

Comment:
If you're gonna have phoneNumber as null, why add it at all?

```
const updatedAdmin = {
    email: adminEmail
}
```
would work fine as far as I see.